src/accordion-item/AccordionItem.js

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+import { usePrevious } from 'react-hanger';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import scrollIntoViewIfNeeded from 'smooth-scroll-into-view-if-needed';
+
+import Heading from './components/Heading';
+import Content from './components/Content';
```

Comment:
Should `components` be renamed to `modules`? @paulobmarcos

Reply:
The `modules` folder is very next-with-moxy specific for shared components. In this case, both are very attached to the `AccordionItem` so `components` suits the purpose well :+1: